internal/docker.go

```diff
@@ -105,6 +105,11 @@ func findDockerContainerIAMRole(container *docker.Container, request *Request) (
 	return "", fmt.Errorf("Could not find IAM_ROLE in the container ENV config")
 }
 
+func findDockerContainerExternalId(container *docker.Container, request *Request) (string, error) {
```

Comment:
I would drop the `error` return in this function, since we just discard it anyway :) 

```suggestion
func findDockerContainerExternalId(container *docker.Container, request *Request) string {
```